vendor/k8s.io/kubernetes/cmd/kubelet/app/server.go

```diff
@@ -94,6 +94,8 @@ import (
 	"k8s.io/kubernetes/pkg/util/rlimit"
 	"k8s.io/kubernetes/pkg/version"
 	"k8s.io/kubernetes/pkg/version/verflag"
+
+	"github.com/openshift/origin/pkg/cmd/server/origin"
```

Comment:
You can't have this dependency. kube depending on origin will make a cycle we can't reasonably resolve during a rebase.

Reply:
opened https://github.com/openshift/origin/pull/19822 as a stop-gap to catch these